### DISCUSSION TITLE
PickerInput getRowOptions isDisabled does not affects appearance

### DISCUSSION THREAD

**Discussion by oleksapro** [epam/UUI#577] [Q&A] [answered]
# Description

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Pass in PickerInput getRowOptions={() => ({ isDisabled: true })}
2. Open PickerInput's DropDown
3. Check rows

## Actual result

Rows looks like enabled, and we can select them.

https://codesandbox.io/s/uui-pickerinput-isdisabled-row-nsor5o?file=/Example.tsx

## Expected result

Rows looks like disabled (gray) and we can't select them.

# Environment

loveship

* UUI version: 4.6.3
* Browser name: Chrome
* Operating System: Windows

**Comment by AlekseyManetov** [comment, marked as answer]
Hi, it's also required to provide `isSelectable: false`, because we are merging your `getRowOptions` config with ours and it's needed to change isSelected field value.
Look on this example - https://uui.epam.com/documents?id=pickerInput&mode=doc&skin=UUI4_promo&category=components#setting_row_options

**Reply by oleksapro** [reply]
Got it, thank you @AlekseyManetov for response.
I'd like give you some context, why I thought it is issue.
When I use isDisabled: true and isSelectable: false the row looks the same like active rows and it is confusing our users. Could we reconsider appearance of disabled dropdowns' rows and make them grayer? 😅

**Reply by AlekseyManetov** [reply]
<img width="695" alt="image" src="https://user-images.githubusercontent.com/14283670/172421365-3e6d761b-eeef-489b-8a03-9692cf50518c.png">
I didn't quite catch you. This is appearance which i got when set isDisabled: true and isSelectable: false, do you have another one? 

Also, you can edit this codesandbox and provide your code example - https://codesandbox.io/s/d6pm8g?file=/Example.tsx

**Reply by oleksapro** [reply]
No, it looks exactly the same for me) "Blade" is disabled, but it is difficult to see difference between disabled row and enabled.

The main problem is disabled row looks' like enabled with small text color change. 

We would like see disabled rows similar to:

![image](https://user-images.githubusercontent.com/7101211/172445893-add11775-cdca-4022-9aaa-4c744024e5f9.png)

**Reply by AlekseyManetov** [reply]
Except the text color it also don't have hover effect and have another pointer. Anyway if we want to change the disabled state appearance we need to confirm it with our design team and update our specifications. Can your project designer contact with Dzmitry Trubchyk and discuss this moments?
